frontend/app/views/offer/subscriber.scala.html

```diff
@@ -0,0 +1,93 @@
+@import model.Faq
+@import com.gu.membership.salesforce.Tier
+@import model.Benefits
+@import views.support.Prices._
+
+@main("Subscribers") {
+    @defining(Benefits.details(Tier.Partner)) { benefits =>
+        <main role="main">
+
+            <div class="page-slice page-slice--slim l-constrained l-side-margins">
+            @fragments.offer.callout("images/offer/subscriptions/audience-1140.jpg", "Pay nothing for 6 months")
+            </div>
+
+            <section class="tone-white l-gutter l-constrained">
```

Comment:
Not 100% about the tone-white class it's coupled a little but to the colour. Could this not also be a `.page-slice`?